<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowCalendar</class>
 <widget class="QDialog" name="WindowCalendar">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendario</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/calendar-icon_34471.png</normaloff>img/calendar-icon_34471.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="gridVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
